src/components/pages/Login/LoginContainer.js

```diff
@@ -21,7 +25,7 @@ const LoginContainer = () => {
       // add your custom logo to your signing/register widget here.
       i18n: {
         en: {
-          'primaryauth.title': 'Welcome to Labs Basic SPA Please sign in',
+          'primaryauth.title': 'WELCOME BACK',
```

Comment:
For people that are interested to know how to style okta widget checkout 
https://developer.okta.com/docs/guides/custom-widget/main/#modify-strings